analyse_heterogeneous_sequences.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+from dark.fasta import FastaReads
+from math import log10
+from heterogeneous_sequences import heterogeneousSites, compareToRef
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--homogenFraction", type=float, default=1.0,
+                    help="If the fraction of the most common nucleotide at a site is at least this value, the site will be considered homogenous.")
+parser.add_argument("--reference",
+                    help="Filename of the reference fasta sequence.")
+args = parser.parse_args()
+
+if 0 < args.homogenFraction < 1:
+    raise ValueError('--homogenFraction needs to be between 0 and 1.')
+
+reads = list(FastaReads('Alignment-CavV-strains.fasta'))
```

Comment:
In a more general program you'd make this be a command-line option too, or have the program read from standard input (which can be extremely useful / convenient - e.g., `grep XXXX file.fasta | your-program.py`).